### PR TITLE
Example Python3.5 support

### PR DESCRIPTION
**Pull request by halfenif**
Example Python3.5 support

change print(f"{}") style to print("{}".format(value))

### CODE REVIEW

File: examples/events.py [hunk 1]
```diff
@@ -58,4 +61,4 @@
         loop.run_until_complete(demo(docker))
     finally:
         loop.run_until_complete(docker.close())
-        loop.close()
+loop.close()
```
Comment (achimnol) [inline examples/events.py]: Please preserve the indent here!

File: examples/stdio_stdout.py [hunk 2]
```diff
@@ -58,4 +58,4 @@
         loop.run_until_complete(demo(docker))
     finally:
         loop.run_until_complete(docker.close())
-        loop.close()
+loop.close()
```
Comment (achimnol) [inline examples/stdio_stdout.py]: Please preserve the indent here!

File: examples/events.py [hunk 3]
```diff
@@ -30,22 +30,25 @@
...
 
     while True:
         event = await subscriber.get()
         if event is None:
             break
-        print(f"event: {event!r}")
+
+        for key, value in event.items():
+            print(key,':', value)
+        #print("event: {}".format(event!r))
```
Comment (achimnol) [inline examples/events.py]: Please clean up the debug comments and update the coding style to add a space after commas (e.g., `key,':',` => `key, ':'`)

File: examples/info.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+from aiodocker.docker import Docker
+from aiodocker.exceptions import DockerError
+
+
+async def demo(docker):
+    print('--------------------------------')
+    print('- Check Docker Version Information')
+    data_version = await docker.version()
+    for key, value in data_version.items():
+        print(key,':', value)
```
Comment (achimnol) [inline examples/info.py]: Needs a space after comma.

File: examples/info.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+    print('--------------------------------')
+    print('- Check Docker Image List')
+    images = await docker.images.list()
+    for image in images:
+        for key, value in image.items():
+            if key == 'RepoTags':
+                print(key,':', value)
```
Comment (achimnol) [inline examples/info.py]: Needs a space after comma.

File: examples/info.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+    print('--------------------------------')
+    print('- Check Docker Container List')
+    containers = await docker.containers.list()
+    for container in containers:
+        container_show = await container.show()
+        for key, value in container_show.items():
+            if key == 'Id':
+                print('Id',':', value[:12])
```
Comment (achimnol) [inline examples/info.py]: Needs a space after comma.

File: examples/events.py (hunk 3 above)
Comment (halfenif) [inline examples/events.py]: fixed

File: examples/events.py (hunk 1 above)
Comment (halfenif) [inline examples/events.py]: fixed

File: examples/stdio_stdout.py (hunk 2 above)
Comment (halfenif) [inline examples/stdio_stdout.py]: fixed

File: examples/info.py (hunk 4 above)
Comment (halfenif) [inline examples/info.py]: fixed

File: examples/info.py (hunk 5 above)
Comment (halfenif) [inline examples/info.py]: fixed

File: examples/info.py (hunk 6 above)
Comment (halfenif) [inline examples/info.py]: fixed

**Pull request merged**